frontend/src/types/card.type.ts

```diff
@@ -0,0 +1,41 @@
+export interface ICard {
+        name: string  
+        description: string
+        address: string
+        regularPrice: number 
+        discountPrice: number
+        bathrooms: number
+        bedrooms: number
+        furnished: boolean
+        parking: boolean
+        type: string
+        offer: boolean
+        imageUrls:string,
+        area: number,
+        userRef: string,
+}
+
+export interface IForYouHome {
```

Comment:
gộp 2 props này vào thành 1
```suggestion
export interface ICard{
        isOpen?: boolean,
        name: string,
        regularPrice: number,
        area: number,
        address: string,
        imageUrls: string,
        save: boolean,
        date?: string,
}
```